.github/workflows/ci.yaml

```
@@ -213,12 +190,15 @@ jobs:
 
       - name: Integration tests
         run: vendor/bin/pest tests/Integration
+        env:
+          DB_PORT: ${{ job.services.database.ports[3306] }}
```

actionlint (1.7.7)

property access of object must be type of string but got "number"

(expression)